### ISSUE TITLE
x-axis data missing

### ISSUE THREAD

**Issue by seriouslylost** [makerdao/maker-market#131]
hi,

https://maker-market.surge.sh/#trade/W-ETH/MKR.
The chart on the right hand side could use information on the x-axis. dates etc 
Sorry if this has been mentioned before but I saw nothing when flipping through the outstanding issues.

**Comment by Rune23** [comment]
yeah, the depth chart right now is pretty funny. Distance between points aren't determined by price distance, but simply by number of orders (regardless of volume or price of those orders). This should be fixed alongside the new price chart getting added

**Comment by ferni** [comment]
Distance between points should be fixed by this PR: https://github.com/OasisDEX/oasis-pro/pull/213
